test/event/handlers/CodeblocksOverFileUploadsHandlerTest.ts

```diff
@@ -41,15 +40,23 @@ describe("CodeblocksOverFileUploadsHandler", () => {
 
 			expect(addMock.calledOnce).to.be.false;
 		});
-		
+
 		it("does nothing when there is a valid attachment.", async () => {
 			message.attachments.set("720390958847361064", new MessageAttachment("720390958847361064", "test.png"));

```

Comment:
I thought that maybe we could have that string as a constant at the top of the file?